perfscale_regression_ci/scripts/common.sh

```diff
@@ -138,4 +138,94 @@ function uncordon_all_nodes() {
   for worker in ${worker_nodes}; do
     oc adm uncordon $worker
   done
+}
+
+# pass $namespace $deployment_name $initial_pod_num $final_pod_num
+# e.g. delete_project "test=concurent-job"
+function check_deployment_pod_scale()
+{
+	namespace=$1
+	deployment_name=$2
+	initial_pod_num=$3
+	final_pod_num=$4
+
+	# Sometimes, it takes a while for pods to scale (up or down). Decrement the counter each time we chack for the pods (in 
+	# the deployment). The pods should scale before count_scaling reaches a negative value, but if it does become negative, 
+	# give an error message and exit the test. This same logic follows for count_running. It takes some time for the pods to 
+	# terminate (scale down) or start running (scale up). The pods should all be in a Running state before count_running 
+	# reaches a negative value. If count_running ecome negative, give an error message and exit the test.
+	count_scaling=1000
+	count_running=1000
```

Comment:
I think this number is way too high. If we set this to 1000 and then wait for 3 seconds inbetween each iteration I think it would be waiting for 50 minutes. I dont think it would ever take that long.

Reply:
The count should never reach this number. I chose this number b/c there are times where the pods scale very slowly (nothing to do with the test, it happens randomly). I wanted to make sure that the this part of the test did not fail if the pods scaled slower than normal.